content/community/_index.html

```diff
@@ -8,9 +8,9 @@ <h1> Community</h1>
   <h2> Mailing List</h2>
 
   <p>
-    General questions or comments for the Git community can be sent to the mailing list by using the email address <a href="mailto:git@vger.kernel.org">git@vger.kernel.org</a>. 
+    General questions or comments for the Git community can be sent to the mailing list by using the email address <a href="mailto:git@vger.kernel.org">git@vger.kernel.org</a>.
   </p>
-  
+
```

Comment:
83d71aaa07419fd8e2c4ada146160b385fa9cb2e:
I'm not sure these whitespace adjustments really are "how things are done with Hugo".

Reply:
Right, this one should have been squashed into the previous commit. I've done that now.

Reply:
This is the combined commit: abd602ef0